driverVersions.json

```diff
@@ -113,6 +117,10 @@
         "64": {
             "driverNPMPackageVersion": "1.14.1",
             "driverVersion": "0.23.0"
+        },
+        "65": {
+            "driverNPMPackageVersion": "1.14.1",
+            "driverVersion": "0.23.0"
```

Comment:
did you mean to use 0.24.0 here?

https://github.com/mozilla/geckodriver/releases

Reply:
@cdagli ping